content/blog/2019-05-22-ramlegacy.md

```diff
@@ -0,0 +1,51 @@
+
+In this post, we’ll talk a little bit about the package development process we went through for `ramlegacy`.
+
+## Package Development Process
+
+When we first started developing `ramlegacy` in Sep 2018, the different versions of the database were all hosted on a Wordpress website created by the RAM maintainers. The package back then was using that Wordpress website as download source for all those versions. However, shortly after v4.3 was released the older versions (v1.0, v2.0, v2.5, v3.0) became unavailable on the Wordpress website. We believed it was important for the package to continue providing access to the aforementioned older versions because, among other reasons, it would enable the users to reproduce dozens of high profile papers based on those older versions; for example [this one](https://doi.org/10.1126/science.aau1758) recently appearing in Science uses the v3.0 data. Consequently, we decided to host the older versions of the database at a GitHub assets [repo](https://github.com/kshtzgupta1/ramlegacy-assets) making sure we appropriately acknowledged and licensed the data.
+
+As we realized how important a stable data source was for the package’s robustness we began a dialog with the RAM Maintainers about moving all the versions of the database to a more permanent DOI-backed repository. Soon after, the RAM Team released three new versions (v4.40, v4.41, v4.44) on Zenodo, a general-purpose open-access repository developed under the European OpenAIRE program and operated by CERN. Zenodo allows researchers to deposit data sets, research software, reports, and any other research related digital artifacts for free. Moreover, it supports [doi versioning](https://blog.zenodo.org/2017/05/30/doi-versioning-launched/) which makes it perfect for a database like RAM Legacy which has multiple versions. Since any data you submit to Zenodo is stored in the CERN Data Center and in the highly unlikely event of its closing Zenodo guarantees that it will migrate all content to other suitable repositories, it is a very stable and reliable data source. The RAM maintainers’ decision to migrate to Zenodo was a great step in ensuring constant access to the RAM Legacy data. However the migration process is not all up to speed yet and all the versions older than v4.4 are not yet on Zenodo. Ideally, the older versions would also be on Zenodo so that the package has a direct official line to the data. Based on our discussions with them, we are hopeful the RAM Legacy team will eventually post those, but it is hard to know exactly when. In the meantime while  the package now downloads the newer versions from Zenodo DOIs it has to continue relying on the backup GitHub repo for the older versions.
```

Comment:
```suggestion
As we realized how important a stable data source was for the package’s robustness we began a dialogue with the RAM Maintainers about moving all the versions of the database to a more permanent DOI-backed repository. Soon after, the RAM Team released three new versions (v4.40, v4.41, v4.44) on Zenodo, a general-purpose open-access repository developed under the European OpenAIRE program and operated by CERN. Zenodo allows researchers to deposit data sets, research software, reports, and any other research related digital artifacts for free. Moreover, it supports [DOI versioning](https://blog.zenodo.org/2017/05/30/doi-versioning-launched/) which makes it perfect for a database like RAM Legacy which has multiple versions. Since any data you submit to Zenodo is stored in the CERN Data Center and in the highly unlikely event of its closing Zenodo guarantees that it will migrate all content to other suitable repositories, it is a very stable and reliable data source. The RAM maintainers’ decision to migrate to Zenodo was a great step in ensuring constant access to the RAM Legacy data. However the migration process is not all up to speed yet and all the versions older than v4.4 are not yet on Zenodo. Ideally, the older versions would also be on Zenodo so that the package has a direct official line to the data. Based on our discussions with them, we are hopeful the RAM Legacy team will eventually post those, but it is hard to know exactly when. In the meantime while  the package now downloads the newer versions from Zenodo DOIs it has to continue relying on the backup GitHub repository for the older versions.
```

Suggested change that can be accepted as a commit:
repo -> repository
doi -> DOI
dialog -> dialogue